### PR TITLE
Add version info to report a bug template

### PR DESCRIPTION
**Pull request by Sherlouk**
Not sure what message you want in the template, so happy to change!

Related to #130 

Output is like this:

Example

### CODE REVIEW

**Comment by Sherlouk** [comment]
Maybe also not a bad idea to add OS version, device name?

**Comment by rnystrom** [comment]
!!!!!! @Sherlouk you rock! I didn't know this was possible!

> Maybe also not a bad idea to add OS version, device name?

Ya let's do it! Wanna add it into this PR?

**Comment by Sherlouk** [comment]
Can do!

Are you alright with the model name Swift spits out? "iPhone8,1" nonsense, or do you want an extension to turn it to something helpful?

**Comment by Sherlouk** [comment]
@rnystrom 

> I didn't know this was possible!

Can do other things like `title`, `labels`, `assignee` etc (obviously depending on perms)

File: Classes/Utility/UIDevice+Model.swift [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        case "iPad4,7", "iPad4,8", "iPad4,9":           return "iPad Mini 3"
+        case "iPad5,1", "iPad5,2":                      return "iPad Mini 4"
+        case "iPad6,7", "iPad6,8":                      return "iPad Pro"
+        case "AppleTV5,3":                              return "Apple TV"
+        case "i386", "x86_64":                          return "Simulator"
+        default:                                        return identifier
+        }
+    }
+    
+}
```
Comment (rnystrom) [inline Classes/Utility/UIDevice+Model.swift]: so good!

File: Classes/Settings/SettingsReportSectionController.swift [hunk 2]
```diff
@@ -31,10 +31,28 @@ final class SettingsReportSectionController: ListSectionController {
...
+        }
+        
+        if let buildNumber = Bundle.main.infoDictionary?["CFBundleVersion"] as? String {
+            builder += " (\(buildNumber))"
+        }
+        
+        builder += "\nDevice: \(UIDevice.current.modelName) (iOS \(UIDevice.current.systemVersion))"
+        
+        return builder + "\n"
+    }
```
Comment (rnystrom) [inline Classes/Settings/SettingsReportSectionController.swift]: this is awesome

**Pull request merged**